security/hosts_multitenancy.adoc

```diff
@@ -8,68 +8,50 @@
 :toc: macro
 :toc-title:
 :prewrap!:
-:sourcedir: ../
 
 toc::[]
 
-[[security-hosts-the-os-matters]]
-== The OS Matters
+[[security-hosts-how-containers-are-secured-on-rhel]]
+== How Containers are Secured on RHEL
 
-Containers make it easier for developers to build and promote the application
-and its dependencies as a unit. Containers also make it easy to get the most use
-of your servers by simplifying multi-tenancy deployments. You can easily deploy
+Containers enable you to simplify multi-tenancy deployments by deploying
 multiple applications on a single host, using the kernel and the docker runtime
-to spin up each individual container.
+to spin up each container.
 
-To take full advantage of this packaging and deployment technology, an
-operations team needs the right environment for running containers. You need an
-operating system (OS) that can secure the host kernel and secure containers from
-each other.
+You must have an operating system (OS) that can secure the host kernel and
+secure containers from each other. Because {product-title} runs on Red Hat
+Enterprise Linux (RHEL) and RHEL Atomic Host, the following concepts apply by
+default to any deployed {product-title} cluster and are at the core of what make
+containers secure on the platform.
 
-Because {product-title} runs on Red Hat Enterprise Linux (RHEL) and RHEL Atomic
-Host, the following concepts apply by default to any deployed {product-title}
-cluster and are at the core of what make containers secure on the platform.
+In Linux, containers are just a special type of process, so securing containers
+is the same as securing any running process. Containers should run as a non-root
+user. Dropping the privilege level or creating containers with the least amount
+of privileges possible is recommended.
 
-[[security-hosts-how-containers-are-secured-on-rhel]]
-== How Containers are Secured on RHEL
+The following security features are available to secure containers running on
+RHEL and RHEL Atomic Host:
```

Comment:
To make this flow a little better, i'd suggest the above be deleted, and replaced with the "Because OCP runs on RHEL and RHEL Atomic..." sentence from a few lines up. It talks about the "following features", and the above lines are sort of a repeat of this, so I think it's workable.